### PR TITLE
feat: navigation bar

### LINKED ISSUE #73: Menubar

### PR DESCRIPTION
**Pull request by izabellalarsson**
Implement a bottom navigation bar to the application.

### CODE REVIEW

File: packages/app/package.json [hunk 1]
```diff
@@ -13,10 +13,22 @@
   "dependencies": {
     "@apollo/client": "^3.0.0-beta.24",
     "@apollo/link-ws": "^2.0.0-beta.3",
+    "@react-native-community/masked-view": "^0.1.6",
+    "@react-navigation/bottom-tabs": "^5.0.0",
+    "@react-navigation/native": "^5.0.0",
+    "@react-navigation/stack": "^5.0.0",
     "graphql": "^14.5.8",
+    "install": "^0.13.0",
```
Comment (momentiris) [inline packages/app/package.json]: Whats going on here? :)

Comment (momentiris) [reply packages/app/package.json]: Do we need "npm" and "install" as dependencies or is this a mistake?

**Review by momentiris — COMMENTED** [review]
Looks good. Left some questions!

File: packages/app/package.json (hunk 1 above)
Comment (izabellalarsson) [reply packages/app/package.json]: def a mistake. Will fix!

**Comment by momentiris** [comment]
Nice

**Review by momentiris — APPROVED** [review]

**Pull request merged**